docs/resources/cloud_credential.md

```diff
@@ -120,3 +120,29 @@ The following attributes are exported:
 - `create` - (Default `10 minutes`) Used for creating cloud credentials.
 - `update` - (Default `10 minutes`) Used for cloud credential modifications.
 - `delete` - (Default `10 minutes`) Used for deleting cloud credentials.
+
+## Import
+
+Node Template can be imported using the Rancher Node Template ID
+
+```bash
+terraform import rancher2_cloud_credential.foo &lt;cloud_credential_id&gt;.&lt;driver&gt;
+```
+
+### Argument Reference
+
+The following arguments are supported:
+
+* `cloud_credential_id` - (Required) The ID of the Cloud Credential (string)
+* `driver` - (Required) The driver of the Cloud Credential (string)
+
```

Comment:
These are not really terraform arguments, moreover import parameters (all required). Could you please use a format like https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/app_v2#import , including the parameters at the import definition??

Reply:
I Removed the arguments part and just created a list.